packages/keyring-snap-bridge/src/SnapKeyring.test.ts

```diff
@@ -225,8 +186,10 @@ describe('SnapKeyring', () => {
 
   // Fake the ControllerMessenger and registers all mock actions here:
   const controllerMessenger: ControllerMessenger<
-    SnapKeyringAllowedActions,
-    SnapKeyringAllowedEvents
+    // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
+    SnapKeyringActions | AllowedActions,
+    // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
+    SnapKeyringEvents | AllowedEvents
```

Comment:
Eslint needs to be disabled here since `SnapKeyringActions` and `AllowedEvents` are both set to `never` meaning they'll get overridden by the other union's members.

I wanted to keep the union syntax here, since that's what we use in every other controllers.

Reply:
I'm on the fence here. I think we should just remove the `never` types and add an union in the future if needed. WDYT?